gdx-controllers-desktop/src/main/java/com/badlogic/gdx/controllers/desktop/support/JamepadController.java

```diff
@@ -164,7 +167,7 @@ private void updateButtonsState() {
 
     private void initializeState() {
         for (ControllerAxis axis : ControllerAxis.values()) {
```

Comment:
I know this is only called once, but is there a reason this does not use `CONTROLLER_AXIS_VALUES`?

Reply:
Exactly, I was only concerned with the update method. Since this is only called once, it doesn't matter here. I'll happily change it to what you prefer.

Reply:
I think it is better to use the new constant, otherwise code readers may ask why it is used only at some places.

Reply:
Alright. Won't get to it this weekend, but will implement the changes after the weekend.